public/app/features/alerting/unified/utils/datasource.ts

```diff
@@ -218,6 +225,7 @@ export function getExternalRulesSources(): ExternalRulesSourceIdentifier[] {
   return getRulesDataSources().map((ds) => ({
     name: ds.name,
     uid: ds.uid,
+    ruleSourceType: 'external',
```

Comment:
We have rule origins of `grafana` and `datasource`, but this type is `grafana` and `external` - should we be consistent/make it talk about "origin" here instead?